client/app/scripts/charts/nodes-layout.js

```diff
@@ -402,7 +402,7 @@ function cloneLayout(layout, nodes, edges) {
 function copyLayoutProperties(layout, nodeCache, edgeCache) {
   const result = Object.assign({}, layout);
   result.nodes = layout.nodes.map(node => (nodeCache.has(node.get('id'))
-    ? node.merge(nodeCache.get(node.get('id'))) : node));
+    ? nodeCache.get(node.get('id')).merge(node) : node));
```

Comment:
I see, that also aligns much better with the method name and its jsdoc. Updated!